src/components/YearSelection.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from 'reactstrap';
+
+const DropdownMenuStyle = {
```

Comment:
Let's use `camelCase` for variable/constant names.

Reply:
done